Tetris_formal.py

```diff
@@ -1,10 +1,14 @@
-import pygame
+""" 
```

Comment:
""" This is the code relating to Tetris with the blocks displaying pictures or colors."""